scripts/create-cohort-project.js

```diff
@@ -91,6 +91,19 @@ const copy = async (src, repoDir, opts) => {
   }
 };
 
+const addBootcampInfo = async (repoDir) => {
+  const projectPkgJsonPath = path.resolve(`${repoDir}/package.json`);
+  let pkg = require(projectPkgJsonPath);
+  let last_commit_hash = (await exec("git rev-parse HEAD")).stdout.replace(/(\r\n|\n|\r)/gm, "");
+  let bootcamp_info = {
+    created_at: Date.now(),
+    bootcamp_version: process.env.npm_package_version,
+    last_bootcamp_commit: last_commit_hash
+  }
+  pkg.bootcamp_info = bootcamp_info;
+  await fs.writeFile(projectPkgJsonPath, JSON.stringify(pkg, null, 2));
```

Comment:
Siguiendo el _estilo_ ya existente en este repo, propondría algo así...

```suggestion
  const pkg = Object.assign(require(projectPkgJsonPath), {
    bootcamp: {
      createdAt: (new Date()).toISOString(),
      version: process.env.npm_package_version,
      commit: (await exec('git rev-parse HEAD')).stdout.trim(),
    },
  });
  await fs.writeFile(projectPkgJsonPath, JSON.stringify(pkg, null, 2));
```

Además de los cambios a nivel de estilo, cambiaría la representación de `createdAt` para que sea interpretable tanto a ojo por un ser humano como para la máquina (cadena ISO, por ejemplo `2022-10-27T16:15:15.268Z`).

:information_source: El método `String.prototype.trim` ya se encarga de quitar _whitespace_ alrededor de un _string_ (del principio y final).

:warning: El código de arriba lo he escrito directamente en el comentario y no lo he probado todavía :see_no_evil:

Reply:
Gracias @lupomontero me parecen bien, me gusta tener el mismo estilo. A ver si @icarito quiere agregar algo mas y probar eso o si no hacemos y probamos nosotros.